examples/3d/deferred_rendering.rs

```diff
@@ -0,0 +1,410 @@
+//! This example compares Forward, Forward + Prepass, and Deferred rendering.
+
+use std::f32::consts::*;
+
+use bevy::{
+    core_pipeline::{
+        fxaa::Fxaa,
+        prepass::{DeferredPrepass, DepthPrepass, MotionVectorPrepass, NormalPrepass},
+    },
+    pbr::{CascadeShadowConfigBuilder, DirectionalLightShadowMap},
+    pbr::{DefaultOpaqueRendererMethod, NotShadowCaster, OpaqueRendererMethod},
+    prelude::*,
+    render::render_resource::TextureFormat,
+};
+
+fn main() {
+    App::new()
+        .insert_resource(Msaa::Off)
+        .insert_resource(DefaultOpaqueRendererMethod(OpaqueRendererMethod::Deferred))
+        .insert_resource(ClearColor(Color::rgb_linear(0.05, 0.05, 0.05)))
+        .insert_resource(AmbientLight {
+            color: Color::WHITE,
+            brightness: 1.0 / 5.0f32,
+        })
+        .insert_resource(DirectionalLightShadowMap { size: 4096 })
+        .add_plugins(DefaultPlugins)
+        .insert_resource(Normal(None))
+        .insert_resource(Pause(true))
+        .add_systems(Startup, (setup, setup_parallax))
+        .add_systems(
+            Update,
+            (animate_light_direction, switch_mode, spin, update_normal),
+        )
+        .run();
+}
+
+fn setup(
+    mut commands: Commands,
+    asset_server: Res<AssetServer>,
+    mut materials: ResMut<Assets<StandardMaterial>>,
+    mut meshes: ResMut<Assets<Mesh>>,
+) {
+    commands.spawn((
+        Camera3dBundle {
+            camera: Camera {
+                //hdr: true,
```

Comment:
?

Reply:
Woops, just was there for testing. I think I put a comment that it works both ways.